Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -1840,7 +1840,9 @@ inline OptionSet<HangingPunctuation> BuilderConverter::convertHangingPunctuation
 
 inline GapLength BuilderConverter::convertGapLength(BuilderState& builderState, const CSSValue& value)
 {
-    return (value.valueID() == CSSValueNormal) ? GapLength() : GapLength(convertLength(builderState, value));
+    if (!value.isPrimitiveValue())
+        return { };
+    return (value.valueID() == CSSValueNormal) ? GapLength() : convertLength(builderState, value);
```

Comment:
Parenthesis around value.valueID() == CSSValueNormal seems unnecessary.

Reply:
It was there in the original code. I can get rid if it if improves readablity

Reply:
Modified it anyway as per your suggestion